amundsen_application/static/js/components/SearchPage/styles.scss

```diff
@@ -2,7 +2,7 @@
 
 .search-page {
   display: flex;
-  min-height: calc(100% - #{$nav-bar-height + $footer-height});
+  min-height: calc(100vh - #{$nav-bar-height + $footer-height});
```

Comment:
Yeah, height: 100% is tricky as it needs all its containers up to the <html> tag to have height 100% as well!